Wox.Infrastructure/StringMatcher.cs

```diff
@@ -8,14 +8,16 @@ namespace Wox.Infrastructure
 {
     public static class StringMatcher
     {
+        public static MatchOption EmptyMatchOption = new MatchOption();
```

Comment:
Extremly picky on my part but I think it will convey a better meaning, so I apologise in advance if I offend- could we change this to DefaultMatchOption instead of EmptyMatchOption. It does till have ignore case set as true.

Reply:
NP changed